nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/resources/docs/org.apache.nifi.processors.standard.PutEmail/additionalDetails.html

```diff
@@ -0,0 +1,111 @@
+<!DOCTYPE html>
+<html lang="en" xmlns="http://www.w3.org/1999/html">
+<!--
+      Licensed to the Apache Software Foundation (ASF) under one or more
+      contributor license agreements.  See the NOTICE file distributed with
+      this work for additional information regarding copyright ownership.
+      The ASF licenses this file to You under the Apache License, Version 2.0
+      (the "License"); you may not use this file except in compliance with
+      the License.  You may obtain a copy of the License at
+          http://www.apache.org/licenses/LICENSE-2.0
+      Unless required by applicable law or agreed to in writing, software
+      distributed under the License is distributed on an "AS IS" BASIS,
+      WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+      See the License for the specific language governing permissions and
+      limitations under the License.
+    -->
+
+<head>
+    <meta charset="utf-8"/>
+    <title>PutEmail</title>
+    <link rel="stylesheet" href="/nifi-docs/css/component-usage.css" type="text/css"/>
+</head>
+<body>
+
+<h1>OAuth Authorization Mode</h1>
+
+<p>
+    PutEmail can use OAuth2. The exact way may depend on the email provider.<br><br>
+</p>
+
+<h2>OAuth with Gmail</h2>
+
+<h3>Configure Gmail OAuth Client</h3>
+
+<p>
+The Gmail OAuth client can be used to send email on behalf of multiple different gmail accounts so this needs to be done once.
+<ol>
+    <li>In the Google Development Consone <a href="https://support.google.com/googleapi/answer/6251787">Create a project</a> (if you don't have one yet)</li>
+    <li><a href="https://console.cloud.google.com/apis/credentials/consent">Configure OAuth consent</a></li>
```

Comment:
I had to make the app external (testing mode) to make it work. When the app was internal, it was not working when doing the next steps. It may be worth mentioning it.

Reply:
Internal apps should work with GMail accounts within the same organization. This could be dependent on the sometimes complicated enterprise account management of Google.

Reply:
Thanks @tpalfy - let's keep things as-is then 👍🏻